layouts/partials/blog/sidebar.html

```diff
@@ -2,31 +2,62 @@
     Blog sidebar, showing a list of tags, and a quick intro for context.
 -->
 <aside class="md:pr-8">
-    <h3 class="no-anchor">
+
+    <!-- Sidebar menu toggle for mobile. -->
+    <div class="md:hidden">
+        <button class="blog-sidebar-toggle items-center px-3 py-2 border rounded text-purple border-purple">
+            <i class="fas fa-bars"></i>
+        </button>
+        <a href="{{relref . "/blog" }}">🍹 Pulumi Blog</a>
+    </div>
+
+    <h3 class="no-anchor hidden md:flex mt-4">
         <a href="{{relref . "/blog" }}">🍹 Pulumi Blog</a>
     </h3>
 
-    <div class="bg-gray-100 rounded p-4 my-8 text-gray-700">
-        <h5 class="no-anchor text-gray-700">Program the Cloud</h5>
-        <p class="text-sm my-0">
-            Create, deploy, and manage cloud infrastructure using your favorite language.
-        </p>
-    </div>
+    <!-- Sidebar content. Hidden on smaller displays unless toggled. -->
+    <div class="blog-sidebar-content hidden md:block">
+        <div class="bg-gray-100 rounded-lg p-4 my-6 text-gray-700">
+            <h5 class="no-anchor text-gray-700">Program the Cloud</h5>
+            <p class="text-sm my-0">
+                Create, deploy, and manage cloud infrastructure using your favorite language.
+            </p>
+        </div>
 
-    <ul class="list-none p-0 md:mr-8">
-        {{ $title := .Title | lower }}
-        {{ range $tag, $_ := .Site.Taxonomies.tags }}
-            {{ if $tag }}
-                <li class="inline-flex">
-                    {{ $classes := "bg-white border border-gray-200 text-blue-600 hover:border-blue-600" }}
-                    {{ if eq $tag $title }}
-                        {{ $classes = "border-blue-600 bg-blue-600 text-white" }}
-                    {{ end }}
-                    <a class="py-1 px-2 text-sm rounded transition-all {{ $classes }}" href="{{ .Page.Permalink }}">
-                        {{ $tag }}
-                    </a>
-                </li>
+        <ul class="list-none p-0 mb-4 inline-flex flex-wrap text-xs">
+            {{ $title := .Title | lower }}
+            {{ range $tag, $_ := .Site.Taxonomies.tags }}
+                {{ if $tag }}
+                    <li>
+                        {{ $classes := "bg-white border border-gray-200 text-blue-600 hover:border-blue-600" }}
+                        {{ if eq $tag $title }}
+                            {{ $classes = "border-blue-600 bg-blue-600 text-white hover:text-white" }}
+                        {{ end }}
+                        <a class="px-2 py-1 mr-1 cursor-auto rounded transition-all {{ $classes }}" href="{{ .Page.Permalink }}">
```

Comment:
Why `cursor-auto`?

I was playing around with the site with this change applied, and clicking the tags felt off without the regular pointer cursor for links.

![tags](https://user-images.githubusercontent.com/710598/60297151-74b8b580-98dc-11e9-92fc-3b4291d11425.gif)

Reply:
Here's what it'd look like with standard pointer cursor for links, which feels better to me:

![tags2](https://user-images.githubusercontent.com/710598/60297332-cd884e00-98dc-11e9-8f8c-7b43c29aa914.gif)

Reply:
Oh, no! That was a mistake on my part. I'll fix. Thanks for noticing it.

Reply:
https://github.com/pulumi/docs/pull/1245